static/src/stylesheets/module/experiments/_acquisitions-epic-design-variations.scss

```diff
@@ -0,0 +1,71 @@
+@import "svg";
+
+// Classes used by multiple variants
+// =================================
+
+.contributions__title--epic--large {
+    @include font-size(36px);
+    @include mq(tablet) {
+        @include font-size(38px);
+    }
+}
+
+// 1. extra_paragraph variant
+// ==========================
+
+// copy changes only
+
+// 2. large hook variant
+// =====================
+
+// uses large title
+
+// 3. subtle variant
+// =================
+
+.contributions__epic--subtle {
+    // Unset border top as it will be set for the inline header element
+    border-top: 0;
+    background-color: #ffffff;
+    margin-top: 50px;
+    margin-bottom: 50px;
```

Comment:
Oh, I forgot the sumit my comment for those: please use `$gs-baseline * 4` instead of `50px`. Then I'll stop nitpicking :)